Browser/keywords/interaction.py

```diff
@@ -297,6 +297,10 @@ def click(
         during the click, and then restores current modifiers back.
         If not specified, currently pressed modifiers are used.
         """
+        if self.library.presenter_mode:
+            self.hover(selector)
+            self.library.highlight_elements(selector, duration=timedelta(seconds=2))
+            sleep(2)

```

Comment:
Why we need sleep here and in the other places?

Reply:
To make highlight happen before the click, text fill, typing, closing happens.